src/admin/store/actions.ts

```diff
@@ -0,0 +1,7 @@
+import { MenuItem } from '../types';
+import { MenuActionTypes } from './types';
+
+export const setMenuItems = (menuItems: MenuItem[]) => ({
```

Comment:
moet je zelf je redux store schrijven? Graphql bewaard deze toch in de graphql cache?
https://hackernoon.com/how-graphql-replaces-redux-3fff8289221d

Reply:
Fixed